<launch>

	 <node name="qbo_face_following" pkg="qbo_face_following" type="qbo_face_following" output="screen" >
	 </node>
	  <param name="/qbo_face_following/move_base" value="true" />
	 <param name="/qbo_face_following/search_min_pan" value="-0.7" /> 
	 <param name="/qbo_face_following/search_max_pan" value="0.7" /> 
	 <param name="/qbo_face_following/search_pan_vel" value="0.3" />  
	 <param name="/qbo_face_following/search_min_tilt" value="-0.7" /> 
	 <param name="/qbo_face_following/search_max_tilt" value="-0.5" />
     <param name="/qbo_face_following/search_tilt_vel" value="0.3" /> 
     
     <param name="/qbo_face_following/desired_distance" value="1.5" /> 

	 
</launch>

